kubeconfig.go

```diff
@@ -123,6 +123,8 @@ func (k *KubeConfig) getKubeConfigFromSecret(ctx context.Context, secretName, se
 	}
 	if bytes, ok := secret.Data["kubeConfig"]; ok {
 		return bytes, nil
+	} else if bytes, ok := secret.Data["value"]; ok {
```

Comment:
Great to add the CAPI support but what about adding comments for why we need to support both keys?

Reply:
Good call, done :)